pkg/volume/flexvolume/unmounter.go

```diff
@@ -42,15 +42,15 @@ func (f *flexVolumeUnmounter) TearDown() error {
 }
 
 func (f *flexVolumeUnmounter) TearDownAt(dir string) error {
-
 	pathExists, pathErr := mount.PathExists(dir)
-	if !pathExists {
-		klog.Warningf("Warning: Unmount skipped because path does not exist: %v", dir)
-		return nil
-	}
-
-	if pathErr != nil && !mount.IsCorruptedMnt(pathErr) {
-		return fmt.Errorf("Error checking path: %v", pathErr)
+	if pathErr != nil {
```

Comment:
how about 
if !pathExists & pathErr != nil {
    return nil
}
...

Reply:
then this function would depend on return value of PathExists, if it return <false, error>, it won't proceed, mainly I would like to cherry pick to old release, only for flexvol.TearDownAt func

Reply:
only if it return (false, nil) which means path is definitely not exist , it will return immediately without any further actions. Isn't this behavior what we want to have here?

Reply:
Hi @jingxu97 logic of the original PR is identical to this PR:
```
	if pathErr != nil {
		// only log warning here since plugins should anyways have to deal with errors
		klog.Warningf("Error checking path: %v", pathErr)
	} else {
		if !pathExists {
			klog.Warningf("Warning: Unmount skipped because path does not exist: %v", dir)
			return nil
		}
	}
```

Reply:
yes, the logic is the same.  I am thinking to make it easier to read. And I think the other PR  #75645 also uses the simpler way.

Reply:
@jingxu97 shall we go with this PR first, and I would like to cherry pick this PR to old release, thanks.